Symfony/CS/Fixer/Symfony/PhpdocShortDescriptionFixer.php

```diff
@@ -13,6 +13,7 @@
 
 use Symfony\CS\AbstractFixer;
 use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
```

Comment:
@GrahamCampbell no it does not. Use statements are used at compile time to resolve names, not at runtime AFAIK